docs/dunst.5.pod

```diff
@@ -240,16 +240,12 @@ is equivalent to
 Defines width in pixels of frame around the notification window. Set to 0 to
 disable.
 
-=item B<gaps> (values: [true/false], default: false)
+=item B<gap_size> (default: 0)
 
-If set to true, display a gap of size B<gap_size> between notifications.
+Size of gap to display between notifications.
```

Comment:
Maybe mention that when gap is 0, separator lines are used and when gap is bigger, a frame is used